packages/client/manifest.json

```diff
@@ -4566,6 +4566,45 @@
       }
     ]
   },
+  "grid": {
+    "name": "Grid",
```

Comment:
We should make this Grid (Beta) as we discussed, just to make it more clear.

Reply:
Will do. Wasn't sure what way we had left it after the few changes to the scope and spec.